docs/guides/integration-prebid-client-side.md

```diff
@@ -133,6 +133,8 @@ pbjs.setConfig({
 }); 
 ```
 
+It is possible that the user has opted out of EUID previously. In this case, EUID module respects the user's optout and no EUID token is generated and collected by Prebid.js.
```

Comment:
... In this case, EUID module respects the user's optout ...
Update to
... In this case, the EUID module respects the user's optout ...

Reply:
Made this edit in branch for update per Sunny's request.